nss-day-cohort-52/giffygram-nunchuck-scorpions
# this should fix our database issue

#### Changes Made

1. Modified the files in our api directory to include `giffygram.json` and removed the unnecessary example database .
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cs-createpost
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
 please check your api directory and make sure that we now have the correct giffygram.json file and this should be the only file in our api directory (full disclosure, there is more code that was changed in createpost.js as well as giffygram.js, the code that was changed is just implementing html representation of the create a post form 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
